README.md

```diff
@@ -193,7 +191,7 @@ instructions for other platforms, and just in general, we recommend you see [Ras
 
 1. Make your target from the build directory you created.
       ```sh
-      $ make hello_world
+      $ cmake --build build --target hello_world
```

Comment:
...or we should be consistent and just use the long command-line switches? (which are "more self-documenting" than the short options)